src/ralph/back_office/admin.py

```diff
@@ -79,6 +81,9 @@ class BackOfficeAssetAdmin(
         'status', 'barcode', 'purchase_order', 'model', 'user', 'warehouse',
         'sn', 'hostname', 'invoice_date', 'invoice_no', 'region',
     ]
+    multiadd_fields = ['sn', 'barcode']
```

Comment:
true. So:
* leave it as it is right now
* in the next PR add additional BO Asset fields (https://github.com/allegro/ralph_assets/blob/develop/src/ralph_assets/models_assets.py#L1062) and add imei to multiadd_fields only if condition is fulfilled (https://github.com/allegro/ralph_assets/blob/develop/src/ralph_assets/forms.py#L750)